SPECS/cert-manager/cert-manager.spec

```diff
@@ -58,6 +59,7 @@ Webhook component providing API validation, mutation and conversion functionalit
 
 %prep
 %setup -q -a 1
+%autopatch -p1
```

Comment:
We can use the collective %autosetup -p1 macro.

Reply:
@Sumynwa thank you, I've changed it now.